README.md

```diff
@@ -115,12 +122,14 @@ github_release and github_actions_secret in the github_provider!) -->
     docker-compose --env-file=.backend.secrets run --rm terraform workspace select ${ENV_NAME}
     ```
 
-1. Run Terraform apply, review the plan, and answer `yes` when prompted.
+1. Run Terraform plan and review the output
 
     ```bash
-    docker-compose --env-file=.env.${ENV_NAME}.secrets run --rm terraform apply -var-file=terraform.${ENV_NAME}.tfvars
+    docker-compose --env-file=.env.${ENV_NAME}.secrets run --rm terraform plan -var-file=terraform.${ENV_NAME}.tfvars
     ```
 
+1. If everything looks good, modify the above command to run Terraform apply. Review the plan again, and answer `yes` when prompted.
```

Comment:
Let's make this just a bit more explicit to make sure it's clear. 🙂  (I'm not sure if the formatting will be correct with the suggestion here)

```suggestion
1. If everything looks good, run this command:

    ```bash
    docker-compose --env-file=.env.${ENV_NAME}.secrets run --rm terraform apply -var-file=terraform.${ENV_NAME}.tfvars
    ````
   Review the plan again, and answer `yes` when prompted.
```